### ISSUE TITLE
synthetic example: rather use single event waveforms

### ISSUE THREAD

**Issue by gesape** [HerrMuellerluedenscheid/pinky#2]
...it is just much too slow to generate synthetic continuous data. I think it would be okay to have an example for the locations only, skipping the detector...

**Comment by HerrMuellerluedenscheid** [comment]
Yeah, the detector logic should be improved/changed. However, can you pull pyrocko's master branch and add a `continuous: false` to the scenario's waveform generator? That should now skip any pure noise traces and hopefully speed things up a little.

**Comment by gesape** [comment]
okay, i will let you know how long it takes now.
